documentation/decisions/README.md

```diff
@@ -44,10 +44,13 @@ While architecture decisions come in all shapes and sizes, some common examples
 When an architectural decision needs to be recorded, please use the following steps:
 
 1. [Create a new ADR ticket in GitHub](https://github.com/HHS/grants-equity/issues/new/choose)
-2. Identify the approvers for this decision, i.e. individuals or groups who must sign off on the decision before the ADR can be accepted (ideally this happens before work starts on the ADR)
+2. Identify the approvers for this decision, i.e. individuals or groups who must sign off on the decision before the ADR can be merged (ideally this happens before work starts on the ADR)
 3. Draft an ADR in a branch named after the ticket, then open a Pull Request (PR)
+  - Use the `{YYYY-MM-DD}-{description}.md` naming convention and copy the [template](./template.md) into the [adr](./adr/) directory (name the file with the date it was created, which may differ from the date the decision was made, PR was merged, or file was last modified)
```

Comment:
> The date refers to the date drafted, merged, or last updated?

Thinking more on this, I've updated the guidance to use the date when you create the file. That's most awkward when we're making ADRs retroactively, but it's the most straightforward method.